<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CategoryListDialogClass</class>
 <widget class="QWidget" name="CategoryListDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Category</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="5">
    <widget class="QTableView" name="tableView">
     <property name="cursor" stdset="0">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="buttonAdd">
     <property name="icon">
      <iconset resource="Money.qrc">
       <normaloff>:/Images/edit_add.png</normaloff>:/Images/edit_add.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="buttonDel">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="icon">
      <iconset resource="Money.qrc">
       <normaloff>:/Images/edit_remove.png</normaloff>:/Images/edit_remove.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="buttonEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="icon">
      <iconset resource="Money.qrc">
       <normaloff>:/Images/Notepad2.png</normaloff>:/Images/Notepad2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="buttonClose">
     <property name="icon">
      <iconset resource="Money.qrc">
       <normaloff>:/Images/editdelete.png</normaloff>:/Images/editdelete.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Money.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>CategoryListDialogClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
